src/mfpbench/lcbench_tabular/benchmark.py

```diff
@@ -187,3 +191,89 @@ def __init__(
             prior=prior,
             perturb_prior=perturb_prior,
         )
+
+    @property
+    def fidelity_range(self) -> tuple[int, int, int]:
+        return (1, 51, 1)
+        
+    def get_raw_space(self, name: int | None = None, seed: int | None = None) -> ConfigurationSpace:
```

Comment:
Hmmm, I'm not sure how to approach this actually. I would prefer this to be a property of the benchmark but of course it's going to be specific to each tabular benchmark, which some may not have:

My suggestion that minimizes type difference is:
* Classic `Benchmark` will always have a `.space: ConfigurationSpace`, as it is currently.
* `TabularBenchmark` will also always have a `.space`. They will be required to either pass in an **empty** `ConfigurationSpace` if there is none available, otherwise they pass in whatever space they can associate with it (i.e. as you've done here).

Changes:
* This will remove the current version of `.space` for `TabularBenchmark` which currently is just categorical of ids. These can still be access through `self.configs: dict[config_id, Config]` or through `.config_keys`

This design serves two purposes:
* Someone using the `GenericTabularBenchmark` interface can pass in the raw space they want to incorporate and it will act like any prebuilt benchmark.
* Keep behaviour more uniform, i.e. there's less chance people do weird funky shit in their own overridden `get_raw_space()`. They either pass in a space or not and we can handle logic for that in one place.